layers/+lang/javascript/packages.el

```diff
@@ -260,6 +261,38 @@
           "sR" 'spacemacs/skewer-eval-region-and-focus
           "ss" 'skewer-repl)))))
 
+(defun javascript/init-indium ()
+  (when (eq javascript-repl 'indium)
```

Comment:
Good idea, but slightly wrong place, please move it into the `use-package if` directive.